cmd/snap-confine/snap-confine-invocation.c

```diff
@@ -61,13 +61,17 @@ void sc_init_invocation(sc_invocation *inv, const struct sc_args *args, const ch
         die("cannot run with NULL executable");
     }
 
+    char snap_name[53] = {0};
```

Comment:
Please define a constant for the 53 and document where it is coming from. I think we have those elsewhere already.

Reply:
I vaguely recall we had this discussion around the time the code for splitting instance name was added, but I don't remember why we didn't add defines back then

Reply:
Constants added